src/etc/cargo.bashcomp.sh

```diff
@@ -154,8 +154,15 @@ _cargo()
 					local opt_var=opt__${cmd//-/_}
 				fi
 				if [[ -z "${!opt_var-}" ]]; then
-					# Fallback to filename completion.
-					_filedir
+					# Forward to subcommands completion if bash-completion >= 2.12 is available
```

Comment:
For future reference:

* https://github.com/scop/bash-completion/blob/2f87ac492c375fd2a3a76a087fcaf92e363f911a/bash_completion#L26-L30